src/Core/x/xstransvc.h

```diff
@@ -32,7 +37,7 @@ enum en_LAYER_TYPE
 	PKR_LTYPE_CUTSCENETOC,
 	PKR_LTYPE_JSPINFO,
 	PKR_LTYPE_NOMORE,
-    PKR_LTYPE_ALL = 0xffffffff
+	PKR_LTYPE_ALL = 0xffffffff
```

Comment:
Are you sure you've run clang-format according to  our rules? I don't think we allow tabs